pulumi/infra/config.py

```diff
@@ -213,6 +213,10 @@ def require_artifact(artifact_name: str) -> Any:
     return artifact
 
 
+def get_postgres_instance_class() -> str:
```

Comment:
Looks like other similar functions don't have the `get_` prefix; consistency would be good.

Also, should we make these all properties? 🤔 (EDIT: oh, these are all just bare functions; I was thinking we had an object. Refactoring for another time, I suppose.)

Reply:
this was actually removed and replaced fwiw

Reply:
```
@dataclass
class PostgresConfigValues:
    instance_type: str
    postgres_version: str

    @staticmethod
    def from_config() -> PostgresConfigValues:
        return PostgresConfigValues(
            instance_type=pulumi.Config().require("postgres-instance-type"),
            postgres_version=pulumi.Config().require("postgres-version"),
        )
```